velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1797,3 +1800,17 @@ TEST_F(StringFunctionsTest, concatInSwitchExpr) {
       {"This is a long sentence-zzz"_sv, "aaa-This is some other sentence"_sv});
   test::assertEqualVectors(expected, result);
 }
+
+TEST_F(StringFunctionsTest, varbinaryLength) {
+  VectorPtr vector;
+  BaseVector::ensureWritable(
+      SelectivityVector(3), VARBINARY(), pool_.get(), vector);
+  auto flat = vector->asFlatVector<StringView>();
+  flat->set(0, "hi");
```

Comment:
Why not use `makeFlatVector<std::string>({"hi", ""}, VARBINARY())`?

Reply:
i did not we can do that.